cpp/src/Ice/Network.h

```diff
@@ -42,14 +42,10 @@ typedef int ssize_t;
 #elif (defined(__APPLE__) || defined(__FreeBSD__) || defined(__FreeBSD_kernel__)) && TARGET_OS_IPHONE == 0 &&          \
     !defined(ICE_NO_KQUEUE)
 #    define ICE_USE_KQUEUE 1
-#elif defined(__APPLE__) && !defined(ICE_NO_CFSTREAM)
+#elif defined(__APPLE__)
 #    define ICE_USE_CFSTREAM 1
 #elif defined(_WIN32)
-#    if !defined(ICE_NO_IOCP)
-#        define ICE_USE_IOCP 1
-#    else
-#        define ICE_USE_SELECT 1
```

Comment:
Seems like ICE_USE_SELECT was a fallback for Windows builds with ICE_NO_IOCP. But the fact is that we always build with IOCP on Windows.